dali/pipeline/operators/fused/resize_crop_mirror.h

```diff
@@ -160,15 +160,15 @@ class ResizeCropMirror : public Operator<CPUBackend>, protected ResizeCropMirror
     const TransformMeta &meta = per_thread_meta_[ws->thread_idx()];
 
     // Resize the output & run
-    output->Resize({crop_[0], crop_[1], meta.C});
+    output->Resize({crop_height_[idx], crop_width_[idx], meta.C});
```

Comment:
idx is something different than dataIdx, so this probably does not work as expected.

Reply:
accepted